### ISSUE TITLE
Add an overview of well-known limitations

### ISSUE THREAD

**Issue by lenaschoenburg** [camunda/camunda-docs#1042]
As a developer who sometimes has to support customers and answer questions, I'm missing something that I can link to that lists some well-known limitations of C8. Without such a page, I'm often not sure what these limitations are and how to best explain them to a customer.

Please note that I mean "limitations" in the sense of missing features or features that cause issues in certain conditions, and I don't mean general known "issues" such as a user facing troubleshooting guide, for example "What to do if I can't login".

To kick this off we could start with the following limitations:

1. Long running processes with a large history can cause issues with Operate, Tasklist and Elasticsearch.
2. Zeebe can't schedule more than ~290 years into the future. (Technically Long.MAX_VALUE = 2^63-1 nanoseconds)

**Comment by akeller** [comment]
DevEx can add this to our backlog to incorporate this into the documentation, either into existing pages or create a specific page dedicated to limitations. If there are other known limitations, please add them to the comments.

**Comment by npepinpe** [comment]
@akeller Is this still something you're interested in?

**Comment by akeller** [comment]
> [@akeller](https://github.com/akeller) Is this still something you're interested in?

@npepinpe Yes and no. Most of the feedback I get on documenting limitations is component-specific, and I imagine we'll shift to Orchestration cluster-specific or platform-specific in the future. I would be okay closing this and adding it to specific epic delivery. In fact, I already see this happening with topics like IDP.
